packages/app/src/app/overmind/onInitialize.ts

```diff
@@ -83,7 +83,10 @@ export const onInitialize: OnInitialize = async (
     getCurrentUser: () => state.user,
     onOperationApplied: actions.editor.onOperationApplied,
     onCodeChange: actions.editor.codeChanged,
-    onSelectionChanged: actions.live.onSelectionChanged,
+    onSelectionChanged: selection => {
```

Comment:
```suggestion
    onSelectionChanged: (selection: UserSelection) => {
```

Also import `UserSelection` type

Reply:
This one is not needed, since it's defined in VSCodeOptions